tools/cve/base-images.sh

```diff
@@ -31,35 +33,55 @@ fi
 
 # Hack to get the base images list.
 # We need to figure out the proper way to store this images and avoid template rendering.
-function base_images_tags {
-  base_images=$(grep . $(pwd)/candi/image_versions.yml) # non empty lines
-  base_images=$(grep -v "#" <<< "$base_images") # remove comments
 
-  reg_path=$(grep "REGISTRY_PATH" <<< ${base_images} | awk '{ print $2 }' | jq -r)
+function registryPath() {
+    grep "REGISTRY_PATH" <<<"$base_images" | awk '{ print $2 }' | tr -d '"'
+}
+
+function base_images_tags() {
+  base_images=$(grep . "$(pwd)/candi/image_versions.yml") # non empty lines
+  base_images=$(grep -v "#" <<<"$base_images")            # remove comments
+
+  reg_path=$(grep "REGISTRY_PATH" <<<"$base_images" | awk '{ print $2 }' | tr -d '"')
 
-  base_images=$(grep -v "REGISTRY_PATH" <<< "$base_images") # Not an image
-  base_images=$(grep -v "BASE_GOLANG" <<< "$base_images") # golang images are used for multistage builds
-  base_images=$(grep -v "BASE_RUST" <<< "$base_images") # rust images are used for multistage builds
-  base_images=$(grep -v "BASE_JEKYLL" <<< "$base_images") # images to build docs
-  base_images=$(grep -v "BASE_NODE" <<< "$base_images") # js bundles compilation
+  base_images=$(grep -v "REGISTRY_PATH" <<<"$base_images") # Not an image
+  base_images=$(grep -v "BASE_GOLANG" <<<"$base_images")   # golang images are used for multistage builds
+  base_images=$(grep -v "BASE_RUST" <<<"$base_images")     # rust images are used for multistage builds
+  base_images=$(grep -v "BASE_JEKYLL" <<<"$base_images")   # images to build docs
+  base_images=$(grep -v "BASE_NODE" <<<"$base_images")     # js bundles compilation
 
-  base_images=$(awk '{ print $2 }' <<< "$base_images") # pick an actual images address
-  base_images=$(jq -sr --arg reg "$reg_path" 'map(. | "\($reg)\(.)") | .[]' <<< "$base_images") # "string" -> registry.deckhouse.io/base_images/string
+  base_images=$(awk '{ print $2 }' <<<"$base_images")                                          # pick an actual images address
+  base_images=$(tr -d '"' <<<"$base_images") # "string" -> registry.deckhouse.io/base_images/string
```

Comment:
It can be rewritten with classical pipes:

```
base_images=$(cat "$(pwd)/candi/image_versions.yml" | \
  grep . | \               # Ignore empty lines
  grep -v "#" | \          # Ignore comments
  grep -v "BASE_RUST" | \  # Ignore rust images used for multistage builds
  ...
  awk '{ print $2 }' | tr -d '"' # Pick actual images and clean quotes 
)
```

It is OK to leave it as is, it just looks weird and bash-specific.